README.md

```diff
@@ -1,70 +1,102 @@
-# Getting Started with Create React App
+Calculator
+Simple calculator created with React Framework. It includes all basic operations and edge cases such a number divided by 0 is equal to 'Math Error'.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Calculator
 
-## Available Scripts
+Built With
+Npm
+Heroku
+React
+Live Preview
+Demo
 
-In the project directory, you can run:
+Getting Started
+To get a local copy up and running follow these simple example steps.
 
-### `npm start`
+Prerequisites
+Computer running Mac OS, Windows, or Linux Ubuntu.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Internet browser: Firefox, Google Chrome, Opera, etc...
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+Install NodeJs
+
+Optional (For deployment)
 
-### `npm test`
+Install Heroku CLI
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Setup
+Download the code or copy the git repository to your computer.
+
+To install dependencies you need to run the following command in your terminal.
+
+  npm install
+Available Scripts
+In the project directory, you can run:
 
-### `npm run build`
+npm start
+Runs the app in the development mode.
+Open http://localhost:3000 to view it in the browser.
 
-Builds the app for production to the `build` folder.\
+The page will reload if you make edits.
+You will also see any lint errors in the console.
+
+npm test
```

Comment:
- Learn to format your readme professionally. Check out some tips [here](https://www.makeareadme.com/)